lib/include/tree_sitter/api.h

```diff
@@ -1166,6 +1166,25 @@ uint32_t ts_language_symbol_count(const TSLanguage *self);
 */
 uint32_t ts_language_state_count(const TSLanguage *self);
 
+/**
+ * Get the number of supertype symbols in the language.
+*/
+uint32_t ts_language_supertype_count(const TSLanguage *self);
+
+/**
+ * Get a list of all supertype symbols for the language.
+*/
+const TSSymbol *ts_language_supertypes(const TSLanguage *self);
+
+/**
+ * Get a list of all subtype symbol names for a given supertype symbol.
+*/
+uint16_t ts_language_supertype_map(
```

Comment:
Note: should it be documented that this function returns the amount of subtype symbols? It would be helpful but the downside is that it wouldn't match the rust binding docstring (which obviously has no need to return the length, it just returns a vector)

Reply:
Just edit the Rust docstring then?